tests/test_learner_data.py

```diff
@@ -60,7 +60,8 @@ def setUp(self):
 
     def test_collect_learner_data_no_enrollments(self):
         learner_data = list(self.exporter.collect_learner_data())
-        assert len(learner_data) == 0
+        learner_data_count = len(learner_data)
+        assert learner_data_count == 0
```

Comment:
@snowcrshd Had pylint complained about this? What was the message? (this is really weird, probably false positive)